pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -84,12 +85,12 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 
 		queue := queues.Pop().(*api.QueueInfo)
 
-		// Found "high" priority job
+		// skip the Queue that has no pending job
 		jobs, found := jobsMap[queue.UID]
 		if !found || jobs.Empty() {
 			continue
 		}
-		job := jobs.Pop().(*api.JobInfo)
+		job := jobs.Pop().(*api.JobInfo)	
```

Comment:
```suggestion
		job := jobs.Pop().(*api.JobInfo)
```

Reply:
Thanks for your suggestion, fix has done